gsic/GSIC_Change_Summary_Frag.adoc

```diff
@@ -7,7 +7,7 @@
 
 
 
-* globus-gssapi-gsi-11.16: Add new configuration options
```

Comment:
Another example: Actually these changes are usually meant to be between the prior version and the current version. So why deviate from that approach if we'll have an unmodified copy of the whole 6.0 stuff in the future?